### ISSUE TITLE
Synchronized-BatchNorm-Pytorch

### ISSUE THREAD

**Issue by HyunjunGil** [NVlabs/SPADE#145]
cd models/networks/
git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
cd ../../

Above command is in installation instruction. I think above commands have a problem. If I run third line, the __init__.py of model.networks is overwritten, and thus it causes error 'module 'models.networks' has no attribute 'modify_commandline_options'. How can I solve this problem?

**Comment by HyunjunGil** [comment]
Oh I'm sorry. This is just my fault. Commands have no problem
